report2idea/sshbruteforceml/sshbruteforceml2idea.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/python3
+
+# In case we are in nemea/modules/report2idea/sshbruteforceml and we want to import from repo:
+import os, sys
+sys.path.insert(0, os.path.join(os.path.dirname(__file__), "..", "..", "..", "nemea-framework", "pycommon"))
+
+import argparse
+
+# The whole functionality of reporting is here:
+from report2idea import *
+import socket
+
+
+# Moudle name, description and required input data format
+MODULE_NAME = "sshbruteforceml"
+MODULE_DESC = "Converts output of SSH bruteforce detector based on ML to IDEA."
+
+REQ_TYPE = pytrap.FMT_UNIREC
+REQ_FORMAT = "ipaddr SRC_IP,ipaddr DST_IP,uint16 DST_PORT,uint16 SRC_PORT,uint16 ATTEMPTS,time EVENT_TIME"
+
+# Main conversion function
+def convert_to_idea(rec, opts=None):
+    """
+    Get fields from UniRec message 'rec' and convert it into an IDEA message (Python dict()).
+
+    rec - Record received on TRAP input interface (the report to convert).
+          Its format satisfies what was defined by REQ_TYPE and REQ_FORMAT.
+    opts - options parsed from command line (as returned by argparse.ArgumentParser)
+
+    Return report in IDEA format (as Python dict). If None is returned, the alert is skipped.
+    """
+    source = rec.SRC_IP
+    target = rec.DST_IP
+    failcount = rec.ATTEMPTS
+
+    idea = {
+        "Format": "IDEA0",
+        "ID": getRandomId(),
+        "DetectTime": getIDEAtime(rec.EVENT_TIME),
+        "CreateTime": getIDEAtime(),
+        "Category": [ "Attempt.Login" ],
+        "Description": "Multiple unsuccessful SSH login attempts",
+        "Note": f"Machine Learning model recognized cca {failcount} unsuccessful login attempts from {source} against SSH server {target}",
```

Comment:
"cca" is not used in English as in Czech, native speakers only used it for historical dates (at least accrding to our English teacher :) ), use "approx." or "around" instead.